ligo/haskell/test/Test/Ligo/RegistryDAO.hs

```diff
@@ -60,7 +60,7 @@ test_RegistryDAO =
             proposalMeta = DynamicRec mempty
             proposalSize = metadataSize proposalMeta
             in withSender (AddressResolved wallet1) $ call baseDao (Call @"Propose")
-              (ProposeParams proposalSize proposalMeta)
+              (ProposeParams [mt||] proposalSize [mt||] proposalMeta)
```

Comment:
`MText` inherits its `Monoid` instance from `Text`, so instead of using `[mt||]` I suggest using `mempty` as it's much more readable.